build/Dockerfile

```diff
@@ -12,58 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi9/python-311 as wheel
 
-ARG WHEEL_VERSION=""
-USER root
-RUN --mount=type=cache,target=/root/.cache/pip \
-    python -m pip install --upgrade pip && \
-    python -m pip install build
-COPY tuning tuning
-COPY .git .git
-COPY pyproject.toml pyproject.toml
-# build wheel if wheel version is empty else download the wheel from PyPi
-RUN if [[ -z "${WHEEL_VERSION}" ]]; \
-    then python -m build --wheel --outdir /tmp; \
-    else pip download fms-hf-tuning==${WHEEL_VERSION} --dest /tmp --only-binary=:all: --no-deps; \
-    fi && \
-    ls /tmp/*.whl >/tmp/bdist_name
-
-
-FROM registry.access.redhat.com/ubi9/ubi AS release
-
-ARG CUDA_VERSION=11.8.0
+## Global Args #################################################################
+ARG BASE_UBI_IMAGE_TAG=latest
+ARG BASE_UBI=registry.access.redhat.com/ubi9/ubi
+ARG PYTORCH_INDEX="https://download.pytorch.org/whl"
 ARG USER=tuning
 ARG USER_UID=1000
-ARG SET_NUM_PROCESSES_TO_NUM_GPUS=True
```

Comment:
I think ARG should be assigned to an `ENV SET_NUM_PROCESSES_TO_NUM_GPUS=${SET_NUM_PROCESSES_TO_NUM_GPUS}`

Reply:
Yup I agree that the default in the image is already set to true so this isn't needed. It's a way of building the image with a different setting but since it's an env var that can easily be set at runtime, not needed at build time.